storage/range.go

```diff
@@ -701,7 +701,10 @@ func (r *Range) addWriteCmd(args proto.Request, reply proto.Response, wait bool)
 	if wait {
 		return completionFunc()
 	}
-	go completionFunc()
+	go func() {
+		// error is lost forever :()
```

Comment:
It's not lost forever; it's logged at line 695. We should actually just move that block into this goroutine.